leo_docking/leo_docking/docking_server.py

```diff
@@ -190,8 +41,68 @@ def __init__(self):
         )
 
         # Create and start the introspection server
-        self._smach_introspection_server = smach_ros.IntrospectionServer(
-            "leo_docking", self._state_machine, "/LEO_DOCKING"
+        self._smach_introspection_server = IntrospectionServer(
+            "leo_docking", self._state_machine.state_machine, "/LEO_DOCKING"
+        )
+
+    def _init_ros(self):
+        sub_qos = QoSProfile(
+            reliability=QoSReliabilityPolicy.BEST_EFFORT, durability=QoSDurabilityPolicy.VOLATILE, depth=1
+        )
+        self.create_subscription(ArucoDetection, "/aruco_detections", self._aruco_detection_cb, qos_profile=sub_qos)
+        self.create_subscription(Odometry, "/odometry/filtered/local", self._wheel_odom_cb, qos_profile=sub_qos)
+        self.create_subscription(Float32, "/firmware/battery", self._battery_cb, qos_profile=sub_qos)
+        self.create_subscription(JointState, "/joint_states", self._effort_cb, qos_profile=sub_qos)
+
+        pub_qos = QoSProfile(
+            reliability=QoSReliabilityPolicy.RELIABLE, durability=QoSDurabilityPolicy.VOLATILE, depth=1
+        )
+        self.cmd_vel_pub = self.create_publisher(Twist, "/cmd_vel/nav2", qos_profile=pub_qos)
```

Comment:
Same here, would be nice to default to the regular /cmd_vel

To fix these for our environment we have to remap the topics in our launch files.